libs/esp32boot/esp32init.erl

```diff
@@ -0,0 +1,394 @@
+%
+% This file is part of AtomVM.
+%
+% Copyright 2023 Davide Bettio <davide@uninstall.it>
+%
+% Licensed under the Apache License, Version 2.0 (the "License");
+% you may not use this file except in compliance with the License.
+% You may obtain a copy of the License at
+%
+%    http://www.apache.org/licenses/LICENSE-2.0
+%
+% Unless required by applicable law or agreed to in writing, software
+% distributed under the License is distributed on an "AS IS" BASIS,
+% WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+% See the License for the specific language governing permissions and
+% limitations under the License.
+%
+% SPDX-License-Identifier: Apache-2.0 OR LGPL-2.1-or-later
+%
+
+-module(esp32init).
+
+-export([
+    start/0,
+    start_repl/1,
```

Comment:
Why are the functions after `start/0` even exported?  I can see why handle_req is, but seem the comment below about moving the dev mode logic into its own modules.

Reply:
Some additional utility functions are exported so they can be called from ALISP shell.

Reply:
Great, let’s put them in your dev module then.